vision/style_transfer/README.md

```diff
@@ -0,0 +1,82 @@
+# Fast Neural Style
+
+## Use-cases
+This artistic style transfer model mixes the content of an image with the style of another image. Below are examples of style transfer models. Each model transfers a unique style to a photograph of a door arch.  
+
+<p align="center">
+  <img src="images/input_images/amber.jpg" width="50%" height="50%">
+</p>
+
+|Model        |Mosaic  |Candy  | Rain Princess |Udnie  |Pointilism|
+| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
+|Style Images       |[<img src="images/style_images/mosaic.jpg">](images/style_images/mosaic.jpg)| [<img src="images/style_images/candy.jpg">](images/style_images/candy.jpg)|[<img src="images/style_images/rain_princess.jpg">](images/style_images/rain_princess.jpg)| [<img src="images/style_images/udnie.jpg">](images/style_images/udnie.jpg)| [<img src="images/style_images/pointilism.jpg">](images/style_images/pointilism.jpg)|
+|Output Image      |[<img src="images/output_images/amber_mosaic.jpg">](images/style_images/amber_mosaic.jpg)| [<img src="images/output_images/amber_candy.jpg">](images/style_images/amber_candy.jpg)|[<img src="images/output_images/amber_rain_princess.jpg">](images/style_images/amber_rain_princess.jpg)| [<img src="images/output_images/amber_udnie.jpg">](images/style_images/amber_udnie.jpg)|[<img src="images/output_images/amber_pointilism.jpg">](images/style_images/amber_pointilism.jpg)|
+
+
+## Description
+The model uses the method described in [Perceptual Losses for Real-Time Style Transfer and Super-Resolution](https://arxiv.org/abs/1603.08155) along with [Instance Normalization](https://arxiv.org/pdf/1607.08022.pdf).
+  
+
+## Model
+ |Model        |Download  |MD5 Checksum| Download (with sample test data)|ONNX version|Opset version|
+|-------------|:--------------|:--------------|:--------------|:--------------|:--------------|
+|Mosaic|[6,571 KB](models/mosaic.onnx)  | 39f0d4d12cf758a7aa31eb150d66244a| [7,162 KB](models/mosaic.tar.gz)|1.5.0|9|
+|Candy|[6,571 KB](models/candy.onnx)  | ef6b9b26d2821ee0c082f229b2e6efcd| [7,167 KB](models/candy.tar.gz)|1.5.0|9|
+|Rain Princess|[6,571 KB](models/rain_princess.onnx)  | 8253cf9670bb24b38152bd71de5571f1|[7,173 KB](models/rain_princess.tar.gz)|1.5.0|9|
+|Udnie|[6,571 KB](models/udnie.onnx)  | f3797cf0dd731c83b307ffa76aed2e67| [7,167 KB](models/udnie.tar.gz)|1.5.0|9|
+|Pointilism|[6,571 KB](models/pointilism.onnx)  | e3241660ecd9f14a671d7229bf18cbd1| [7,164 KB](models/pointilism.tar.gz)|1.5.0|9|
+<hr>
+
+## Inference
+Refer to [fns-pytorch-to-onnx.ipynb](fns-pytorch-to-onnx.ipynb) for detailed preprocessing and postprocessing.
+
+### Input to model
+The input to the model are 3-channel RGB images. The images have to be loaded in a range of [0, 255]. If running into memory issues, try resizing the image by increasing the scale number. 
```

Comment:
Could you please double-check that the input images are indeed in RGB format and not anything else like BGR?

Reply:
I double checked and the images are loaded in RGB format.